cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -1024,3 +996,75 @@ func (s *Server) GetInstalledPackageResourceRefs(ctx context.Context, request *c
 		}, nil
 	}
 }
+
+func (s *Server) AddPackageRepository(ctx context.Context, request *corev1.AddPackageRepositoryRequest) (*corev1.AddPackageRepositoryResponse, error) {
+	log.Infof("+helm AddPackageRepository [%v]", request)
```

Comment:
logging the whole request may leak credentials. it would be better to abstract only what is necessary (see how it is done in the method CreateInstalledPackage)

Reply:
It is a line copied from Flux plugin [here](https://github.com/vmware-tanzu/kubeapps/blob/4662-addpackagerepoHelm-2/cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/server.go#L502).
If it is just to locate the operation in the logs, I would say that name and URL should suffice.
@gfichtenholt what do you think?

Reply:
I needed more than name and URL when I was working on flux as some point. I debugged and found and fixed the problem, whatever it was, then and this just kind of remained. I think @dlaloue-vmware is right, could you fix it and also fix one in flux as well if you have the bandwidth. Thanks in advance